compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ReachabilityAnalyzer.java

```diff
@@ -744,15 +744,15 @@ public void visit(BLangLetExpression letExpression, AnalyzerData data) {
         data.statementReturnsPanicsOrFails = returnStateBefore;
     }
 
-    private void checkStatementExecutionValidity(BLangStatement statement, AnalyzerData data) {
+    private void checkStatementExecutionValidity(BLangNode statement, AnalyzerData data) {
```

Comment:
It is better to rename the parameter name too.

Reply:
I will update

Reply:
Updated d40fecc2014d4adb35b8b1117ca9de143e36010